diktat-rules/src/test/kotlin/com/saveourtool/diktat/ruleset/chapter2/KdocMethodsFixTest.kt

```diff
@@ -61,4 +61,10 @@ class KdocMethodsFixTest : FixTestBase("test/paragraph2/kdoc/package/src/main/ko
     fun `KdocMethods rule should reformat code (full example)`() {
         fixAndCompare("KdocMethodsFullExpected.kt", "KdocMethodsFullTested.kt")
     }
+
+    @Test
+    @Tag(WarningNames.KDOC_WITHOUT_THROWS_TAG)
+    fun `@throws tag shouldn't be added`() {
```

Comment:
Add more understandable name for the test. For example, `@throws tag shouldn't be added inside try-catch`

Reply:
Done